crux-llvm/README.md

```diff
@@ -34,16 +34,21 @@ The `crux-llvm` tool can be built by doing the following:
 * Change to the `crux-llvm` directory and run the build script:
 
         cd crucible/crux-llvm
-        ./build-stack.sh
+        cabal v2-build
 
-This will compile `crux-llvm` and supporting libraries and install them
-in the directory `crucible/crux-llvm/bin`.
+This will compile `crux-llvm` and supporting libraries such that they
+can be executed with `cabal v2-run`. To install the binaries in the
+standard Cabal binary path, run the following:
+
+        cabal v2-install exe:crux-llvm --overwrite-policy=always
 
 # Invocation
 
-In the `crux-llvm` directory, to analyze `file.c`, run
+In the `crux-llvm` directory (either in the repository or the root of
+the directory extracted from a distribution tarball), to analyze
+`file.c`, run
```

Comment:
Is there a reason why you have to be in the repository?  Also, you may want to mention that `cruc-llvm` should be in the path.

Reply:
Unfortunately there is, for the moment. The `c-src` directory needs to be a subdirectory of your current directory. We really should fix that, but I don't think it'll happen today.